src/utils/format-time-interval.js

```diff
@@ -0,0 +1,31 @@
+export default function formatTimeInterval (value) {
+	let milliseconds = parseInt(value, 10);
```

Comment:
This function should not convert types. Do it on the parent level. This function should accept only numbers